<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerhitunganAngkatPesawat</class>
 <widget class="QDialog" name="PerhitunganAngkatPesawat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="tinggiTerbang">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="kecTerbang">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="areaSayap">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Koefisien Lift             :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kofLift">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Area Sayap                 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Kecepatan Terbang :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hitung">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Hitung</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tinggi Terbang          : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lift">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelLift">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Lift :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
